examples/flux-source/ngrok/Pulumi.yaml

```
@@ -0,0 +1,39 @@
+name: ngrok
+runtime: yaml
+description: Run ngrok to tunnel webhooks to Flux
+
+variables:
+  appLabels:
+    app: ngrok
+
+resources:
+  ngrok:
+    type: kubernetes:apps/v1:Deployment
+    properties:
+      spec:
+        selector:
+          matchLabels: ${appLabels}
+        replicas: 1
+        template:
+          metadata:
+            labels: ${appLabels}
+          spec:
+            containers:
+              - name: ngrok
+                image: ngrok/ngrok:2.3.40-alpine
+                args:
+                  - http
+                  - webhook-receiver.flux-system:80
+
+  service:
+    type: kubernetes:core/v1:Service
+    properties:
+      spec:
+        selector: ${appLabels}
+        ports:
+          - port: 80
+            targetPort: 4040
+
+outputs:
+  deployment: ${deployment.metadata.name}
```

```suggestion
  deployment: ${ngrok.metadata.name}
```

This needs to match the resource name (not the type 😛)

---

Urk, bad copypasta 🍝